plugins/compiler.js

```diff
@@ -53,8 +53,11 @@
   update node start/end positions of other nodes
   we're interested in */
   function replaceNode(node, newCode) {
+    // TODO: Where is 'code' defined? Is this a global?
```

Comment:
Where is 'code' defined? Is this a global?

Reply:
Eep. It's the parameter in `compileCode` I think - so this function should really be defined inside there at minimum? It's a bit odd that this worked at all!

Reply:
Oh - I spotted why I'd never noticed this as a problem. It was fine before and you 'hoisted' this out of the function last week! https://github.com/rikkuness/EspruinoTools/commit/7af30377d39be77835dec7f042d62cb657da838e :)

Reply:
Damnit, sorry about that! 😬 😂  it was bound to happen sooner or later, hopefully didn't cause too much damage